pkgs/development/libraries/audio/lilv/default.nix

```diff
@@ -14,6 +14,11 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ pkg-config python3 wafHook ];
   buildInputs = [ serd sord sratom ];
   propagatedBuildInputs = [ lv2 ];
+  dontAddWafCrossFlags = true;
+
+  preConfigure = ''
+    export PKGCONFIG="$PKG_CONFIG"
```

Comment:
since waf uses the `PKGCONFIG` variable
https://gitlab.com/ita1024/waf/-/blob/master/waflib/Tools/c_config.py#L213

maybe we should set it in the `wafConfigurePhase` https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/build-managers/wafHook/setup-hook.sh

something like
```bash
if [ -n "${PKG_CONFIG}" ]; then
  export PKGCONFIG="${PKG_CONFIG}"
fi

Reply:
Makes sense to me. I'll try that.

Reply:
Thanks again for the suggestion, I applied it and it seems to work well.